packages/app-js/src/index.tsx

```diff
@@ -75,11 +102,24 @@ class App extends React.PureComponent<Props, State> {
       window: null
     };
 
-    const exec = `(async ({${Object.keys(injected).join(',')}}) => { try { ${code} } catch (error) { console.error(error); } })(injected);`;
+    const exec = `(async ({${Object.keys(injected).join(',')}}) => {
+      try {
```

Comment:
Ahhh, actually made a comment on this here https://github.com/polkadot-js/apps/pull/681/files#diff-7c043ef600c68ef2f18082701888270eR109 (when I saw that one-liner again)

The issue is that the `catch (error) { console.log(error) }` currently suck - we actually want the full trace with line-number and the ability to map-back.

Reply:
You also took care of that in your PR. (At least the 1-liner part)